webapp/src/modules/auction/utils.js

```diff
@@ -1,5 +1,14 @@
+import { env } from 'decentraland-commons'
 import { getLocalStorage } from '@dapps/lib/localStorage'
 
+export const TOKENS = {
+  MANA: env.get('REACT_APP_MANA_TOKEN_CONTRACT_ADDRESS'),
+  ZIL: env.get('REACT_APP_ZIL_TOKEN_CONTRACT_ADDRESS'),
+  DAI: env.get('REACT_APP_DAI_TOKEN_CONTRACT_ADDRESS')
+}
+
+export const SYMBOLS = Object.keys(TOKENS)
```

Comment:
I think this variable might be a bit clearer as `TOKEN_SYMBOLS` when used from outside this file. The first time I saw `import { SYMBOL } from 'modules/auction/utils` I wasn't entirely sure what was the value.
That said it's kind of a big refactor so feel free to dismiss this comment

Reply:
✅ done